lib/radar-rules.js

```diff
@@ -791,7 +791,7 @@ module.exports = {
         hub: [
             {
                 title: '镜像有新 Build',
-                docs: 'https://docs.rsshub.app/program-update.html#docker-hub',
+                docs: 'https://docs.rsshub.app/program-update.html#docker-hub-jing-xiang-you-xin-build',
```

Comment:
Please remove old entry in radar-rules.js

Reply:
The old entry is not in `v2`. Shall I migrate it there?

Reply:
Yes.
See https://github.com/DIYgod/RSSHub/issues/8269